server/protocols/webadmin/webadmin-cassandra/src/test/java/org/apache/james/webadmin/routes/CassandraMigrationRoutesTest.java

```diff
@@ -145,12 +145,23 @@ public void postShouldReturnConflictWhenMigrationOnRunning() throws Exception {
     public void postShouldReturnErrorCodeWhenInvalidVersion() throws Exception {
         when(schemaVersionDAO.getCurrentSchemaVersion()).thenReturn(CompletableFuture.completedFuture(Optional.of(OLDER_VERSION)));
 
-        given()
+        Map<String, Object> errors = given()
             .body(String.valueOf("NonInt"))
         .with()
             .post("/upgrade")
         .then()
-            .statusCode(400);
+            .statusCode(400)
+            .contentType(ContentType.JSON)
+            .extract()
+            .body()
+            .jsonPath()
+            .getMap(".");
+
+        assertThat(errors)
+            .containsEntry("statusCode", 400)
+            .containsEntry("type", "InvalidArgument")
+            .containsEntry("message", "Invalid request for version upgrade")
+            .containsEntry("cause", "For input string: \"NonInt\"");
```

Comment:
`cause : For input string`
doesn't look like a cause ...

Reply:
This string come from a "e.getMessage()" and the "e" in question come from a json parse.
I don't really see what I can do to make that better sorry

Reply:
You described the problem : a user facing message generated by a component we don't manage
Looks like a bad idea